src/main/java/com/dabsquared/gitlabjenkins/GitLabMergeRequest.java

```diff
@@ -60,6 +63,18 @@ public String toString() {
         return ToStringBuilder.reflectionToString(this, ToStringStyle.MULTI_LINE_STYLE);
     }
 
+    public GitlabCommitStatus createCommitStatus(GitlabAPI api, String status, String targetUrl) {
+        try {
+            if(objectAttributes.lastCommit!=null) {
+                return api.createCommitStatus(sourceProject, objectAttributes.getLastCommit().getId(), status, objectAttributes.getLastCommit().getId(), "Jenkins", targetUrl, null);
```

Comment:
According to the [commit status api documentation](http://doc.gitlab.com/ce/api/commits.html#post-the-status-to-commit) for the optional `ref` it says
> ref (optional) - The ref (branch or tag) to which the status refers

But here the commit id is assigned to it again.
Shouldn't it be `objectAttributes.getSourceBranch()`?